src/Send/SendScreen/SendScreen.tsx

```diff
@@ -124,7 +123,7 @@ export const SendScreen = () => {
   }
 
   const handleConfirm = async () => {
-    if (isValid == false || recomputing || yoroiUnsignedTx == null) return
+    if (isValid === false || recomputing || yoroiUnsignedTx == null) return
```

Comment:
```suggestion
    if (!isValid || recomputing) return
```

Reply:
can you tripe-check this one, it looks like `isValid` can be `undefined`, it will produce different results.